cfgov/wagtailadmin_overrides/templates/wagtailadmin/account/password_reset/confirm.html

```diff
@@ -35,7 +35,7 @@ <h1>{% trans "Set your new password" %}</h1>
                     </li>
                 {% endfor %}
                 <li class="submit">
-                    <button type="submit" class="button button-longrunning" tabindex="3" data-clicked-text="Working..."><span class="icon icon-spinner"></span><em>{% trans 'Reset password' %}</em></button>
+                    <button type="submit" class="button button-longrunning" tabindex="3" data-clicked-text="Working..."><span class="icon icon-spinner"></span><em>Reset password</em></button>
```

Comment:
Because this change is only to remove the translation marking around the string within this button, because this comes from an upstream Wagtail template that we've imported and modified, and finally because we're going to be going through this template overrides to ensure they reflect upstream changes as we upgrade to Wagtail 3 and 4, I'm not going to address this in this PR.